### PR TITLE
Add additional MIT LICENSE files to libs

### PR DESCRIPTION
**Pull request by vilkinsons**
# What this does

- Improves the formatting of our root LICENSE file (the underlying license remains unchanged)
- Adds explicit LICENSE files to each of our libs (crates/packages) to assist license compliance checkers in confirming the open-source status of BP components used as dependencies within external repos. Our 3x block templates have been excluded from this (to reduce cruft), but remain MIT-licensed per the top-level LICENSE file's terms.

### CODE REVIEW

File: crates/type-system/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
+=====================
+
+Copyright © 2022 HASH
```
Comment (nonparibus) [inline crates/type-system/LICENSE.md:4]:
```suggestion
Copyright © 2022–, HASH
```

File: packages/@blockprotocol/graph/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
+=====================
+
+Copyright © 2021-2022 HASH
```
Comment (nonparibus) [inline packages/@blockprotocol/graph/LICENSE.md:4]:
```suggestion
Copyright © 2021–, HASH
```

File: packages/@blockprotocol/hook/LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
+=====================
+
+Copyright © 2021-2022 HASH
```
Comment (nonparibus) [inline packages/@blockprotocol/hook/LICENSE.md:4]:
```suggestion
Copyright © 2021–, HASH
```

File: packages/@blockprotocol/type-system-node/LICENSE.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
+=====================
+
+Copyright © 2021-2022 HASH
```
Comment (nonparibus) [inline packages/@blockprotocol/type-system-node/LICENSE.md:4]:
```suggestion
Copyright © 2021–, HASH
```

File: packages/@blockprotocol/type-system-web/LICENSE.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
+=====================
+
+Copyright © 2021-2022 HASH
```
Comment (nonparibus) [inline packages/@blockprotocol/type-system-web/LICENSE.md:4]:
```suggestion
Copyright © 2021–, HASH
```

File: packages/block-scripts/LICENSE.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
+=====================
+
+Copyright © 2021-2022 HASH
```
Comment (nonparibus) [inline packages/block-scripts/LICENSE.md:4]:
```suggestion
Copyright © 2021–, HASH
```

File: packages/blockprotocol/LICENSE.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
+=====================
+
+Copyright © 2021-2022 HASH
```
Comment (nonparibus) [inline packages/blockprotocol/LICENSE.md:4]:
```suggestion
Copyright © 2021–, HASH
```

File: packages/create-block-app/LICENSE.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
+=====================
+
+Copyright © 2021-2022 HASH
```
Comment (nonparibus) [inline packages/create-block-app/LICENSE.md:4]:
```suggestion
Copyright © 2021–, HASH
```

File: packages/mock-block-dock/LICENSE.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
+=====================
+
+Copyright © 2021-2022 HASH
```
Comment (nonparibus) [inline packages/mock-block-dock/LICENSE.md:4]:
```suggestion
Copyright © 2021–, HASH
```

**Review by CiaranMn — DISMISSED** [review]

**Comment by kachkaev** [comment]
I [ran](https://github.com/blockprotocol/blockprotocol/pull/644/commits/55a66db2c2202b3058b922c33cb67fcaedbee13e) `yarn fix:prettier` to fix CI checks. If the change is undesired, we can add license files to `.prettierignore`.

**Review by kachkaev — APPROVED** [review]

**Pull request merged**